sys/vane/ames.hoon

```diff
@@ -1331,6 +1331,8 @@
     ::
         %pito
       :_  fox(tim `p.bon)
+      %-  flop
+      ^-  (list move)
       :-  [gad.fox %pass /ames %b %wait p.bon]
       ?~  tim.fox  ~
       [gad.fox %pass /ames %b %rest u.tim.fox]~
```

Comment:
Would you either stick a comment here, or just reverse the `%wait` and `%rest` moves? I'm a simple man, and having trouble keeping track of >1 flop.

Reply:
Strike this, I don't think we'll forget why this is.